core/modules/messages/server/messages.js

```diff
@@ -66,6 +66,26 @@ const notifyUsers = (channel, message) => {
   log('notifyUsers: done', { userIds });
 };
 
+const notifyUsersInLevel = message => {
+  const { channel, createdBy } = message;
```

Comment:
let's stay consistent with `notifyUsers` and use the same signature, unless there's a real reason to do otherwise?

Reply:
also the two functions are really close and should be refactored IMO

Reply:
Actually, the notifyUsers() function is used only for one & one channel. It should be 'notifyUser' instead. I think, they used this notifyUsers function for zone previously and just let the function like this. But yes, I will merge them both and use only one function.